docs/user_guide/queue_config.md

```diff
@@ -302,3 +302,45 @@ resources:
     <resource name 2>: <0..maxint>
 ```
 Resources that are not specified in the list are not limited, for max resources, or guaranteed in the case of guaranteed resources. 
+
+### Child Template
+
+The parent queue can provide a template to define the behaviour of dynamic leaf queues below it. The supported configuration in template are shown below.
+1. application sort policy
+2. max resources
+3. guaranteed resources
+4. max application
+
+As an example:
+```yaml
+ partitions:
+   - name: default
+     placementrules:
+       - name: provided
+         create: true
+     queues:
+       - name: root
+         submitacl: '*'
+         childtemplate:
+           properties:
+             application.sort.policy: stateaware
+           resources:
+             max:
+               vcore: 20000
+               memory: 600000
+         queues:
+           - name: a0
+             childtemplate:
+               resources:
+                 max:
+                   vcore: 21000
+                   memory: 610000
+             queues:
+               - name: a00
+```
+In this case, the dynamic leaf (`root.b0`) uses the template of `root` to initialize `application.sort.policy`, max resource, and guaranteed resource.
```

Comment:
In this case, the dynamic leaf queue `root.sales`, will use the template set on the parent queue, in this case `root` to....